src/index.html

```diff
@@ -3,11 +3,325 @@
+            <h4 class="recom__description ">portable Bluetooth</h4>
+            <h4 class="recom__description ">$ 149.00</h4>
+          </div>
+
+          <div class="recom__speakers">
+            <div class="recom__image-wrapper">
+              <img 
+                class="recom__image" 
+                src="images/speaker3.png" alt="speaker1"
+              >
+            </div>
+            <h4 class="recom__description ">SoundLink Color Bluetooth speaker II</h4>
+            <h4 class="recom__description ">Portable bluetooth</h4>
+            <h4 class="recom__description ">$ 129.00</h4>
+          </div>
+        </section>
+        <section class="content__category category" id="gallery">
+          <h2 class="category__heading">
+            Browse Bose <br> products by category
+          </h2>
+
+          <div class="category__item category__item--ear">
+            <div class="category__picture-wrapper">
+              <img 
+                class="category__picture" 
+                src="images/category1.png" 
+                alt="girl"
+              >
+            </div>
+            <div class="category__picture-wrapper">
+              <img 
+                class="category__picture" 
+                src="images/category2.png" 
+                alt="girl"
+              >
+            </div>
+            <h3 class="category__description">Headphones & earbuds</h3>
+          </div>
+
+          <div class="category__item category__item--speaker">
+            <div class="category__picture-wrapper">
+              <img 
+                class="category__picture" 
+                src="images/category3.png" 
+                alt="speaker"
+              >
+            </div>
+            <div class="category__picture-wrapper">
+              <img 
+                class="category__picture" 
+                src="images/category4.png" 
+                alt="speaker"
+              >
+            </div>
+            <h3 class="category__description">Speakers</h3>
+          </div>
+
+          <div class="category__item category__item--glass">
+            <div class="category__picture-wrapper">
+              <img 
+                class="category__picture" 
+                src="images/category5.png" 
+                alt="sunglass"
+              >
+            </div>
+            <div class="category__picture-wrapper">
+              <img 
+                class="category__picture" 
+                src="images/category6.png" 
+                alt="sunglass"
+              >
+            </div>
+            <h3 class="category__description">Audio sunglasses</h3>
+          </div>
+          <div class="category__button-wrapper">
+            <input 
+              class="category__button" 
+              type="button" 
+              value="view all"
+            >
+          </div>
+        </section>
+        <section class="content__why why" id="testimonials">
```

Comment:
This time its okay to change padding from margin but you could have given margin to `content__why` class also